### ISSUE TITLE
Cannot create `client` class

### ISSUE THREAD

**Issue by arronhunt** [ptariche/tesla-vehicles-api-ts#3]
```js
const CLIENT = require("tesla-vehicles-api").client;

const TeslaClient = new CLIENT({
  username: "ABCDEFG",
  password: "123456789"
});

// Error: A Username and Password are required or tokens are required
```

**Update:**
https://github.com/ptariche/tesla-vehicles-api-ts/blob/cfbc0481f68c549bf56239669c3c236d2c217044/src/index.ts#L51-L53

It looks like the logic here is faulty. Since providing only a username and password results in `if (false || true)` this statement is always `true` and blocks us from continuing.

**Comment by ptariche** [comment]
Issue resolved #9

**Comment by ptariche** [comment]
0.3.0 should have the resolved issue. the bug was introduced with 4eda6907c8a57960fde09ff404acdbc686323ded
